cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -104,13 +105,21 @@ func (serviceGenerateHelper) formatHandlerFuncName(service *protogen.Service, hn
 
 var helper serviceGenerateHelperInterface = serviceGenerateHelper{}
 
+// Matches the FileDescriptorProto.package field number
```

Comment:
Nit: please terminate comment sentences with a period. Here, and in all other comments added as part of this PR. Thanks.

Reply:
Sorry to disagree on such a meaningless nit here, but this is not a complete sentence, and thus should not be capitalized or end with a period.

Reply:
If you have specific requirements for comments, then please provide the replacement comment that you want.

Reply:
Let's just remove the verb to remove any confusion about grammar: "FileDescriptorProto.package field number".